ios/lib/ReactNativeCameraKit/CKCamera.m

```diff
@@ -461,6 +473,26 @@ + (AVCaptureDevice *)deviceWithMediaType:(NSString *)mediaType preferringPositio
     return captureDevice;
 }
 
+-(void)setTorchMode:(AVCaptureTorchMode)torchMode callback:(CallbackBlock)block
+{
+    _torchMode = torchMode;
+    [CKCamera setTorchMode:torchMode forDevice:self.videoDeviceInput.device];
+    if (block) {
+        block(self.videoDeviceInput.device.torchMode == torchMode);
+    }
+}
+
++(void)setTorchMode:(AVCaptureTorchMode)torchMode forDevice:(AVCaptureDevice *)device
+{
+    if (device.hasTorch && [device isTorchModeSupported:torchMode]) {
+        NSError* err = nil;
+        if ( [device lockForConfiguration:&err] ) {
+            [device setTorchMode:torchMode];
+            [device unlockForConfiguration];
+        }
+    }
+}
+
```

Comment:
Why 2 functions? one is a setter and the second one is public, why not single func?